script/upload.py

```diff
@@ -58,40 +59,55 @@ def main():
           'You have to pass --overwrite to overwrite a published release'
 
   # Upload Electron files.
-  upload_electron(release, os.path.join(DIST_DIR, DIST_NAME), args)
+  # Rename dist.zip to  get_zip_name('electron', version, suffix='')
```

Comment:
this comment should probably read "copy dist.zip to" rather than "rename" (which implies move rather than copy)